libs/@blockprotocol/service/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blockprotocol/service",
-  "version": "0.0.1-canary.9",
+  "version": "0.0.2",
```

Comment:
Do we want to be doing these manually or should `changesets` be handling this for us?

Reply:
It should, be to date this package has been published manually while being developed (which makes sense to get it off the ground). We can let `changesets` take over soon.